data-access/mandataris.ts

```diff
@@ -360,3 +359,47 @@ export async function addLinkToDecisionDocumentToMandataris(
     );
   }
 }
+
+export async function updateStatusOfMandataris(
```

Comment:
Be careful here, we aren't actually updating the status of the mandataris, we are updating the publication status of the mandataris. 
So I would suggest updating the name of this method. And the predicate used in the query should be `extlmb:hasPublicationStatus`